sql/1.096/dynamic-blocks.sql

```diff
@@ -0,0 +1,75 @@
+BEGIN;
+drop table if exists instant_answer_blocks;
+
+create table instant_answer_blockgroup (
+    id serial primary key,
+    blockgroup varchar(20) not null unique
+);
+
+copy instant_answer_blockgroup (blockgroup) from stdin;
+goodie_exlusive
+spice_killresult
+spice_nonexclusive
+goodie
+spice
+fathead
+longtail
+\.
```

Comment:
Can you try again?  I added statements to revert any changes that might have been made as well so you don't need to drop/alter table.  Just `\i` the file.  I was able to repeatedly run it ok.